.github/workflows/sdks_generate_openapi_spec_and_fern_code.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           git add sdks/typescript/src/opik/rest_api/
           git add sdks/code_generation/fern/openapi/openapi.yaml
           git add apps/opik-documentation/documentation/fern/openapi/opik.yaml
-          git commit -m "Update OpenAPI spec and Fern code"
+          git commit --allow-empty -m "Update OpenAPI spec and Fern code"
```

Comment:
don't we have title validation restrictions for PRs to having `[]` in the beginning?